test/workflow-ui/work-items-table.ts

```diff
@@ -42,6 +42,9 @@ const item2 = buildWorkItem(
 const item3 = buildWorkItem(
   { jobID: targetJob.jobID, workflowStepIndex: 2, serviceID: step2ServiceId, status: WorkItemStatus.RUNNING },
 );
+const retryingWorkItem = buildWorkItem(
```

Comment:
See previous comment about naming booleans. I think this would be better as `isRetryingWorkItem`.

Reply:
This one is a work item - not a boolean.